modules/core/src/v2/coins/eth.ts

```diff
@@ -832,6 +845,45 @@ export class Eth extends BaseCoin {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Return types, values, and total amount to send in a batch transaction
+   * @param recipients {Object} transaction recipients
+   * @returns {Object} the types, values, and total amount to execute the batch transaction
+   */
+  static batchExecutionInfo(recipients) {
+    // Method signature is
+    // distributeBatch(address[], uint256[])
+
+    const addresses = [];
+    const amounts = [];
+    let sum = new BigNumber(0);
+    _.forEach(recipients, (recipient) => {
+      addresses.push(recipient.address);
+      amounts.push(recipient.amount);
+      sum = sum.plus(new BigNumber(recipient.amount));
+    });
+
+    return { types: ['address[]', 'uint256[]'], values: [addresses, amounts], totalAmount: sum.toFixed(0) };
+  };
+
+  /**
+   * Changes recipients to point to Batcher contract with correct amount and adds data needed for Batcher execution
+   * @param txPrebuildRecipients {Object} prebuilt transaction recipients
+   * @returns {Object} information necessary to send batch transaction
+   */
+  createBatchParams(recipients) {
+    const { types, values, totalAmount } = Eth.batchExecutionInfo(recipients);
+    const methodSignature = optionalDeps.ethAbi.methodID(Eth.batchMethodName, types);
+    const encodedArgs = optionalDeps.ethAbi.rawEncode(types, values);
+    const batchData = Buffer.concat([methodSignature, encodedArgs]).toString('hex');
```

Comment:
doesn't this need to be 0x prefixed? ethUtil.addHexPrefix

Reply:
In the `getSendData` function in platform's `eth.js` they don't appear to use `ethUtil.addHexPrefix`. They do keep it as a buffer, though, so I'll get rid of `.toString('hex')`.